doc/classes/Environment.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="Environment" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
+<class name="Environment" inherits="Resource" category="Core" version="3.0">
 	<brief_description>
+		Resource for environment nodes (like [WorldEnvironment]) that define multiple rendering options.
```

Comment:
Environment can be used as:
-Inside WorldEnvironment
-Inside a Camera
-As default environment in the Project Settings